<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisWdgOptionsTIFF</class>
 <widget class="QWidget" name="KisWdgOptionsTIFF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>299</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TIFF Options</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>TIFF Options</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Compression type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kComboBoxCompressionType">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>JPEG DCT Compression</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deflate (ZIP)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lempel-Ziv &amp; Welch (LZW)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pixar Log</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="text">
           <string>Predictor:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kComboBoxPredictor">
          <property name="toolTip">
           <string>Using a predictor can improve the compression (mostly for LZW and deflate.)</string>
          </property>
          <property name="currentItem" stdset="0">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Horizontal Differencing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Floating Point Horizontal Differencing</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="alpha">
        <property name="toolTip">
         <string>Disable to get smaller files if your image has no transparency</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;The Portable Network Graphics (PNG) file format allows transparency in your image to be stored by saving an alpha channel.
You can uncheck the box if you are not using transparency and you want to make the resulting file smaller .&lt;br&gt;Always saving the alpha channel is recommended.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Store alpha &amp;channel (transparency)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="flatten">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>This option will merge all layers. It is advisable to check this option, otherwise other applications might not be able to read your file correctly.</string>
        </property>
        <property name="text">
         <string>Flatten the &amp;image</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSaveProfile">
        <property name="text">
         <string>Save ICC Profile</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="codecsOptionsStack">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget1">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame4">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget2">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxJPEG">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>JPEG Compression Options</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>Quality:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QSlider" name="qualityLevel">
              <property name="toolTip">
               <string>These settings determine how much information is lost during compression</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>80</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="textLabel3">
                <property name="text">
                 <string>Smallest</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="textLabel4">
                <property name="text">
                 <string>Best</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget3">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxDeflate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Deflate Compression Options</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel1_4">
            <property name="toolTip">
             <string>Note: the compression level does not change the quality of the result</string>
            </property>
            <property name="whatsThis">
             <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
            </property>
            <property name="text">
             <string>Compress:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QSlider" name="compressionLevelDeflate">
              <property name="toolTip">
               <string>Note: the compression level does not change the quality of the result</string>
              </property>
              <property name="whatsThis">
               <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>6</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="textLabel3_2">
                <property name="whatsThis">
                 <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
                </property>
                <property name="text">
                 <string comment="save the file quickly, but do not compress very well">Fast</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="textLabel4_2">
                <property name="whatsThis">
                 <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
                </property>
                <property name="text">
                 <string comment="make the file small, at the price of a longer saving time">Small</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget5">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxPixarLog">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Pixar Log Compression Options</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="textLabel1_4_2">
            <property name="toolTip">
             <string>Note: the compression level does not change the quality of the result</string>
            </property>
            <property name="whatsThis">
             <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
            </property>
            <property name="text">
             <string>Compress:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QSlider" name="compressionLevelPixarLog">
              <property name="toolTip">
               <string>Note: the compression level does not change the quality of the result</string>
              </property>
              <property name="whatsThis">
               <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>6</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="textLabel3_2_2">
                <property name="whatsThis">
                 <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
                </property>
                <property name="text">
                 <string comment="save quickly, but do not compress a lot">Fast</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="textLabel4_2_2">
                <property name="whatsThis">
                 <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
                </property>
                <property name="text">
                 <string comment="make a very small file, but take a long time saving">Small</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kComboBoxCompressionType</tabstop>
  <tabstop>kComboBoxPredictor</tabstop>
  <tabstop>alpha</tabstop>
  <tabstop>flatten</tabstop>
  <tabstop>qualityLevel</tabstop>
  <tabstop>compressionLevelDeflate</tabstop>
  <tabstop>compressionLevelPixarLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>